app/controllers/admin/books_controller.rb

```diff
@@ -26,7 +27,21 @@ def create
     end
   end
 
+  def destroy
+    respond_to do |format|
+      if @book_delete&.destroy
+        format.js{flash.now[:notice] = t ".book_deleted"}
+      else
+        format.js{flash.now[:danger] = t ".delete_fail"}
+      end
+    end
+  end
+
   private
+  def load_param_destroy
+    @book_delete = Book.find_by(id: params[:id])
```

Comment:
Thêm xử lý khi ko tìm thấy book nữa em
```suggestion
    @book = Book.find_by(id: params[:id])
```